lib/Capabilities.php

```diff
@@ -32,26 +32,18 @@
 use OCP\IConfig;
 use OCP\IUser;
 use OCP\IUserSession;
+use OCP\Translation\ITranslationManager;
 
 class Capabilities implements IPublicCapability {
-	protected IConfig $serverConfig;
-	protected Config $talkConfig;
-	protected ICommentsManager $commentsManager;
-	protected IUserSession $userSession;
-	private IAppManager $appManager;
 
 	public function __construct(
-		IConfig $serverConfig,
-		Config $talkConfig,
-		ICommentsManager $commentsManager,
-		IUserSession $userSession,
-		IAppManager $appManager,
+		protected IConfig $serverConfig,
+		protected Config $talkConfig,
+		protected ICommentsManager $commentsManager,
+		protected IUserSession $userSession,
+		protected IAppManager $appManager,
```

Comment:
This was private `before`.
Is that change on purpose?

Reply:
Yeah, I have protected as it prevents extending